### PR TITLE
Add rds security group

### PR DESCRIPTION
**Pull request by jsoucheiron**
Add a RDSDBSecurityGroup resource.
This PR should be merged after #102

### CODE REVIEW

**Review by w0rmr1d3r — COMMENTED** [review]
Question, won't we create a model for [AWS::RDS::DBSecurityGroupIngress](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-security-group-ingress.html)?
On the other hand, I don't see any test for this new resource either

**Comment by jsoucheiron** [comment]
Honestly, I don't know how [AWS::RDS::DBSecurityGroupIngress](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-security-group-ingress.html) is used, it appears that we're using ingress objects that are simple dictionaries. (While [AWS::RDS::DBSecurityGroupIngress](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-security-group-ingress.html) are full fledged resources.

**Comment by jsoucheiron** [comment]
I've added some really simple tests

File: pycfmodel/model/resources/security_group_ingress.py [hunk 1]
```diff
@@ -47,3 +50,9 @@ def ipv4_slash_zero(self) -> bool:
 
     def ipv6_slash_zero(self) -> bool:
         return self.Properties.ipv6_slash_zero()
+
+
+class RDSDBSecurityGroupIngress(Resource):
+    TYPE_VALUE: ClassVar = "AWS::RDS::DBSecurityGroupIngress"
+    Type: str = TYPE_VALUE
+    Properties: DBSecurityGroupIngressProp
```
Comment (w0rmr1d3r) [inline pycfmodel/model/resources/security_group_ingress.py:58]: Careful, with this, we will miss this parameter [DBSecurityGroupName](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-security-group-ingress.html#cfn-rds-securitygroup-ingress-dbsecuritygroupname).
And we won't be able to model the resource

Comment (jsoucheiron) [reply pycfmodel/model/resources/security_group_ingress.py]: added

**Review by w0rmr1d3r — APPROVED** [review]

**Pull request merged**